support-frontend/assets/pages/digital-subscription-checkout/components/checkoutForm.jsx

```diff
@@ -163,58 +157,16 @@ function CheckoutForm(props: PropTypes) {
           }}
           >
             <FormSection title="Your details">
-              <Input1
-                id="first-name"
-                label="First name"
-                type="text"
-                value={props.firstName}
-                setValue={props.setFirstName}
-                error={firstError('firstName', props.formErrors)}
-              />
-              <Input1
-                id="last-name"
-                label="Last name"
-                type="text"
-                value={props.lastName}
-                setValue={props.setLastName}
-                error={firstError('lastName', props.formErrors)}
-              />
-              <InputWithLabel
-                id="email"
-                label="Email"
-                type="email"
-                disabled
-                value={props.email}
-                footer={(
-                  <span>
-                    <CheckoutExpander copy="Want to use a different email address?">
-                      <p>You will be able to edit this in your account once you have completed this checkout.</p>
-                    </CheckoutExpander>
-                    <CheckoutExpander copy="Not you?">
-                      <p>
-                        <Button
-                          appearance="greyHollow"
-                          icon={null}
-                          type="button"
-                          aria-label={null}
-                          onClick={() => props.signOut()}
-                        >
-                          Sign out
-                        </Button> and create a new account.
-                      </p>
-                    </CheckoutExpander>
-                  </span>
-                )}
-              />
-              <Input1
-                id="telephone"
-                label="Telephone"
-                optional
-                type="tel"
-                value={props.telephone}
-                setValue={props.setTelephone}
-                footer="We may use this to get in touch with you about your subscription."
-                error={firstError('telephone', props.formErrors)}
+              <PersonalDetails
+                firstName={props.firstName}
+                setFirstName={props.setFirstName}
+                lastName={props.lastName}
+                setLastName={props.setLastName}
+                email={props.email}
+                telephone={props.telephone}
+                setTelephone={props.setTelephone}
+                formErrors={((props.formErrors: any): FormError<PersonalDetailsFormField>[])}
```

Comment:
I couldn't find any other way to convince Flow to let me pass the errors array in even though we know that structurally it is ok